mobile-widgets/qmlmapwidgethelper.cpp

```diff
@@ -59,7 +59,22 @@ void MapWidgetHelper::reloadMapLocations()
 
 void MapWidgetHelper::selectedLocationChanged(MapLocation *location)
 {
-	qDebug() << location;
+	int idx;

```

Comment:
38af4120f7139fede5ccdb8272d382895ff46c35 : and another overflowing commit message